package/yast2-core.spec

```diff
@@ -26,7 +26,7 @@
 %bcond_with werror
 
 Name:           yast2-core
-Version:        3.2.2
+Version:        3.3.0
```

Comment:
Why 3.3.0 instead of 3.2.3?

Reply:
well, it is first patch and differentiator between SP3 and SP4. But I think in general it does not matter much.

Reply:
Ok, I am simply asking because this https://github.com/yast/yast-journal/pull/32#discussion_r213303079

I would do exactly what you did, but @lslezak explained me why did not do it. See https://yastgithubio.readthedocs.io/en/latest/versioning/#old-schema